README.md

```diff
@@ -116,29 +129,31 @@ You should see output similar to:
 
     OK
 
-*NOTE*: For all the tests to pass successfully, you\'ll need Postgres (with
+**NOTE!** For all the tests to pass successfully, you\'ll need Postgres (with
 the postgis and hstore extension) & MongoDB to be running locally. You'll
-also need libgeos available.
+also need *libgeos* available.
 
 For Postgres:
-
-    > psql postgres
-    CREATE DATABASE flask_admin_test;
-    \q
-
-    > psql flask_admin_test
-    CREATE EXTENSION postgis;
-    CREATE EXTENSION hstore;
-
+```bash
+psql postgres
+> CREATE DATABASE flask_admin_test;
+> # Connect to database "flask_admin_test":
+> \c flask_admin_test;
+> CREATE EXTENSION postgis;
+> CREATE EXTENSION hstore;
+```
 If you\'re using Homebrew on MacOS, you might need this:
 
-    # install postgis and geos
-    > brew install postgis
-    > brew install geos
+```bash
+# Install postgis and geos
+brew install postgis
+brew install geos
+export DYLD_LIBRARY_PATH=/opt/homebrew/opt/geos/lib/
```

Comment:
We shouldn't need this any more - we were using an older version of geoalchemy, which didn't check this directory. Now it does. Should all of the DYLD_LIBRARY_PATH stuff should be able to go (it's also specific to just darwin/macos).